src/ClusterBootstrap/services/launcher/launcher.yaml

```diff
@@ -19,4 +19,4 @@ spec:
       # serviceAccountName: frameworkcontroller
       containers:
       - name: frameworkcontroller
-        image: frameworkcontroller/frameworkcontroller:v0.5.1
+        image: xudifsd/frameworkcontroller
```

Comment:
We probably should not use personal identifier in repo?

Reply:
Yes.. I also think so.